tedana/workflows/tedana.py

```diff
@@ -375,7 +374,7 @@ def tedana_workflow(data, tes, out_dir='.', mask=None,
     ref_handler.setFormatter(text_formatter)
     RepLGR.setLevel(logging.INFO)
     RepLGR.addHandler(rep_handler)
-    RepLGR.setLevel(logging.INFO)
+    RefLGR.setLevel(logging.INFO)
```

Comment:
Nice catch!

Reply:
Hm, maybe we should make those variables more distinct, or else we'll probably just do this again later.